rule.php

```diff
@@ -435,8 +435,7 @@ private function get_download_button_only() {
      * @return string
      */
     private function get_seb_download_url() {
-        // TODO: Issue #9 - Admin setting or download SEB url.
-        return self::DEFAULT_SEB_DOWNLOAD_URL;
+        return get_config('quizaccess_seb', 'downloadlink');
```

Comment:
If downloadlinkis not set this method will return nothing, then we will render empty button. Probably it's better to check if the setting is empty before rendering the button and if it's empty, then render nothing.

Reply:
Then would be good to test this scenario